openid-4-verifiable-presentations-1_0.md

```diff
@@ -2001,68 +2028,263 @@ Note: The `nonce` and `aud` are set to the `nonce` of the request and the Client
+* Reference: (#error-response) of this specification
+
+### wallet_unavailable
+
+* Name: wallet_unavailable
+* Usage Location: authorization endpoint
+* Protocol Extension: OpenID for Verifiable Presentations
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#error-response) of this specification
+
+## OAuth Authorization Server Metadata Registry
+
+This specification registers the following authorization server metadata parameters
+in the IANA "OAuth Authorization Server Metadata" registry [@IANA.OAuth.Parameters]
+established by [@!RFC8414].
+
+### presentation_definition_uri_supported
+
+* Metadata Name: presentation_definition_uri_supported
+* Metadata Description: Boolean value specifying whether the Wallet supports the transfer of presentation_definition by reference
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#as_metadata_parameters) of this specification
+
+### vp_formats_supported
+
+* Metadata Name: vp_formats_supported
+* Metadata Description: An object containing a list of name/value pairs, where the name is a string identifying a Credential format supported by the Wallet
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#as_metadata_parameters) of this specification
 
-Interoperability considerations: : n/a
+## OAuth Dynamic Client Registration Metadata Registry
 
-- Published specification: : TODO
-- Applications that use this media type: : Applications that issue, present,
-  verify verifier attestation VCs.
-- Additional information:
+This specification registers the following client metadata parameters
+in the IANA "OAuth Dynamic Client Registration Metadata" registry [@IANA.OAuth.Parameters]
+established by [@!RFC7591].
+
+### vp_formats
+
+* Client Metadata Name: vp_formats
+* Client Metadata Description: Boolean value specifying whether the Wallet supports the transfer of presentation_definition by reference
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#client_metadata_parameters) of this specification
+
+## OAuth URI Registry
+
+This specification registers the following URNs
+in the IANA "OAuth URI" registry [@IANA.OAuth.Parameters]
+established by [@!RFC6755].
+
+### urn:ietf:params:oauth:federation
+
+* URN: urn:ietf:params:oauth:federation
+* Common Name: OpenID Federation
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#federations) of this specification
+
+### urn:ietf:params:oauth:federation_trust_mark
+
+* URN: urn:ietf:params:oauth:federation_trust_mark
+* Common Name: OpenID Federation Trust Mark
+* Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
+* Reference: (#federations) of this specification
+
```

Comment:
I am extremely unsure about this. First of all, these values are not defined as URIs in the spec. and I don't think the intent of the section was to define these parameters normatively... and if we are registering these, we should also register `https://train.trust-scheme.de/info`.

Reply:
The time to change the Change Controller is after the specification has been adopted by the DCP working group.

Reply:
Having read the relevant section of OID4VP I agree with Mike here - this does look like the correct outcome of the current spec language. It may well be appropriate to change the values, but we should probably do that in a separate issue if so, it would be out of scope for this PR.

Reply:
I'd argue that it is appropriate to change the values and have #279 with the intent to do so.

This PR can just omit the registry requests so as to not further ingrain the erroneous values and avoid wasting the time of IANA and/or the DEs on that registry. See also https://github.com/openid/OpenID4VP/pull/274#discussion_r1790468552 for related commentary.

I'd also agree with the statement that "these values are not defined as URIs in the spec" - a single passing mention does not constitute definition by a spec. 

Lastly and only tangentially related - https://train.trust-scheme.de/ doesn't seem to resolve to anything and the provided reference for @TRAIN is seemingly to the website for a 2022 conference in Denmark https://oid2022.compute.dtu.dk/index.html, which all seems somewhat less than legitimately use of anything.